root/static/scripts/tests/relationship-editor.js

```diff
@@ -96,7 +96,10 @@ var testRelease = {
                     artistCredit: {names: []},
                 }
             ],
-            format: "Vinyl",
+            format: {
+                name: "Vinyl",
+            },
```

Comment:
Not directly about this code, but: medium formats have MBIDs we seem to not be using at all, or am I missing something? :)

Reply:
I don't think we use them in the code, yet from what I can see. So don't think you're missing anything. :)